.github/workflows/publish.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
I think you are not retrieving this in 

```
      - name: Get yarn cache directory path
        id: yarn-cache-dir-path
        run: echo "::set-output name=dir::$(yarn cache dir)"

      - uses: actions/cache@v3
        id: yarn-cache
        with:
          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-yarn-
```

Reply:
Hi! I'm not too sure if I understand the problem.

This is the execution before the change:
![image](https://user-images.githubusercontent.com/2673520/209793931-fdc9fd5e-92df-4c41-b485-525217488954.png)


This is the execution after the change:
![image](https://user-images.githubusercontent.com/2673520/209793863-489ccd71-beea-4e80-b6fb-8cc6eddece09.png)

It seems the output is being retrieved without problem.

Reply:
Yes, it is actually good. Thanks.